### DISCUSSION TITLE
Change language keyboard

### DISCUSSION THREAD

**Discussion by BeaverFlight** [end-4/dots-hyprland#720] [Q&A]
I can t switch languages through the keys if i enter hyprctl keyboard, the language changes, but it doesnt work through the keyboard and doesnt react at all. In general.conf i prescribed a language change, bu it doesnt work.
input{
    kb_layout = us, ru
    kb_optins = grp:alt_shift_toggle
    numlock_by_default = true
....
}

**Comment by end-4** [comment]
note that hyprland's keyboard layout is per-window, try again
`¯\_(ツ)_/¯`

**Reply by mikhaylovdaniil** [reply]
why u troll him xD

**Comment by konataidzumi** [comment]
ralbno zaebalo
